apps/react-18-tests-v9/config/tests.js

```diff
@@ -0,0 +1,8 @@
+/** Jest test setup file. */
+
+require('@testing-library/jest-dom');
+
+// Clean up created files/folders on exit, even after exceptions
+// (will not catch SIGINT on windows)
+const tmp = require('tmp');
```

Comment:
is this needed for this app ?

Reply:
again, copy over from another package. i'll remove for now.